src/mqtt-client/client.js

```diff
@@ -59,6 +72,16 @@ function onSubscribed() {
             deviceName = topic.split('/').pop();
 
         if (isDeviceInfo) {
+            debug(`Device-info came: ${deviceName}, publishingQueue.size: ${publishingQueue.size}`);
```

Comment:
!fix
if we don't get this message (e.g. when device went off) -- we wont proceed with other requests -- add timeout for discarding request (and moving that to the end of queue)

also consider moving logic with plug-n-play to a separate file